release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4040,4 +4040,23 @@ $ oc adm release info 4.12.32 --pullspecs
 [id="ocp-4-12-32-updating"]
 ==== Updating
 
-To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-33"]
+=== RHBA-2023:5016 - {product-title} 4.12.33 bug fix update
+
+Issued: 2023-09-12
+
+{product-title} release 4.12.33 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:5016[RHBA-2023:5016] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:5018[RHBA-2023:5018] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.33 --pullspecs
+----
+
+[id="ocp-4-12-33-updating"]
+==== Updating
+
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
```

Comment:
```suggestion
To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
```
Consider omitting `for instructions`. It's already clear from the context and not needed, imo.

Reply:
@JoeAldinger , what do you reckon, should I apply this update and change it for all the other release note entries to maintain consistency? I know we have some stock text to follow, but I do agree with Brian than "for instructions" does provide some level of redundancy.

Reply:
This comes up every so often and I don't disagree. However, this is going to require a sweep of 4.11-4.13. Lets merge this as is and we can file a ticket under the CI Jira epic for RNs and take care of the sweep by the end of the week. Sound good?

Reply:
Thanks, Joe. Tracking those updates through https://issues.redhat.com/browse/OSDOCS-7722 is a good idea. Thanks for raising this ticket.